apps/design-land/src/app/app.component.html

```diff
@@ -12,6 +12,7 @@
       <a daff-link-set-item routerLink="accordion">Accordion</a>
       <a daff-link-set-item routerLink="callout">Callout</a>
       <a daff-link-set-item routerLink="card">Card</a>
+      <a daff-link-set-item routerLink="checkbox">Checkbox</a>
```

Comment:
Should be listed under `Atoms`

Reply:
fixed in a19666774303a5986f9a4617acdc5a58ec028acf